src/preferences/dialog/dlgprefsounddlg.ui

```diff
@@ -148,7 +148,7 @@
      <item row="11" column="0">
       <widget class="QLabel" name="masterDelayLabel">
        <property name="text">
-        <string>Master Delay</string>
+        <string>Main Output Delay</string>
```

Comment:
We should either omit Output here or add Output also to Headphone and Booth Delay

Reply:
I think "Main Delay" by itself would be a bit unclear what it is referring to.

Reply:
I'll add "Output" to the rest of the delay labels for consistency.